content/400-reference/200-api-reference/200-command-reference.mdx

```diff
@@ -11,6 +11,36 @@ This document describes the Prisma CLI commands, arguments, and options.
 
 </TopBlock>
 
+## Installation
+
+The Prisma CLI is typically installed as a **development dependency**, that's why the `--save-dev` (npm) and `--dev` (Yarn) options are used in the commands below.
+
+<Admonition>
+
+We **recommend that you install the Prisma CLI locally** in your project's `package.json` to avoid version conflicts.
+
+</Admonition>
+
+### npm
+
+Install with [npm](https://www.npmjs.com/):
+
+```
+npm install prisma --save-dev
+```
+
+### Yarn
+
+```
+yarn add prisma --save-dev
+```
+
+### pnpm
+
+```
+pnpx install prisma --save-dev
+```
+
```

Comment:
maybe we can add the global installation instruction here as an expandable element?